plugins/@grouparoo/mailchimp/src/lib/shared/updateProfile.ts

```diff
@@ -143,8 +143,21 @@ export const updateProfile: UpdateProfileMethod = async ({
   return { success: true };
 };
 
-export async function deleteMember(client, listId, mailchimpId) {
+export async function deleteMember(
+  client,
+  listId,
+  mailchimpId,
+  oldGroups,
+  newGroups
+) {
   try {
+    let tagsToRemove = Array.from(new Set(oldGroups.concat(newGroups)));
+    await client.post(`/lists/${listId}/members/${mailchimpId}/tags`, {
+      tags: tagsToRemove.map((g) => ({
+        name: String(g).toLocaleLowerCase(),
```

Comment:
let's make a funciton in this file for the 3 spots called `normalizeGroupName`
several plugins have that.